lib/backend/model/statusreports.go

```diff
@@ -97,7 +116,13 @@ func (key LastStatusReportKey) defaultDeletePath() (string, error) {
 	if key.Hostname == "" {
 		return "", errors.ErrorInsufficientIdentifiers{Name: "hostname"}
 	}
-	e := fmt.Sprintf("/calico/felix/v1/host/%s/last_reported_status", key.Hostname)
+	if key.RegionString == "" {
+		return "", errors.ErrorInsufficientIdentifiers{Name: "regionString"}
+	}
+	if strings.Contains(key.RegionString, "/") {
```

Comment:
Why implement this validation differently from every other key, which seems to use the validate tag?